bpm-server/src/main/resources/application.yml

```diff
@@ -76,3 +64,7 @@ info.application.version: @project.version@
 digiwf:
   deployment:
     targets: ${BPMSERVER_DEPLOYMENT_TARGETS:DEV, TEST, DEMO, PROCESSESTEST, PROCESSESDEMO, PROCESSESHOTFIX}
+  # Required for the application to start, even though it is not used
+  security:
+    client-id: ${SSO_TASK_CLIENT_ID}
+    client-secret: ${SSO_TASK_CLIENT_SECRET}
```

Comment:
Es wäre ganz cool, wenn wir in der Security Lib per Config den Client aktivieren bzw. deaktivieren können. Der bpm-server ist z.B. ein reiner Resource Server